### ISSUE TITLE
Required parameter `output_prefix` is not necessary in `rseq_inner_distance`

### ISSUE THREAD

**Issue by tverbeiren** [viash-hub/biobox#168]
The variable `output_prefix` is set to `required: true` but it is not really used for the output: https://github.com/viash-hub/biobox/blob/a13b57d04a3f3741eedd1af10fd96a9bee126f55/src/rseqc/rseqc_inner_distance/script.sh#L9

Also, the tests reflect that.

**Comment by DriesSchaumont** [comment]
@tverbeiren I had a look at this issue. I'm interested in your opinion on what the solution should be. AFAIK, the tool itself indeed uses `output_prefix` for two things:
* As a prefix to the files (they are output in the cwd)
* The `Name` column in the stats output
* Some fields in the generated R plots script
 
There are (I think), three options:
- Remove the `output_prefix` argument altogether. The location of the output files are controlled by the other component arguments (e.g. `output_stats` and `output_dist`). However, the ability to control the contents `Name` column in the stats is lost.
- Keep the `output_prefix` argument together with the other output arguments and document that its only used a kind of label.
- Keep the `output_prefix` argument but remove the other output arguments; output a directory instead. This is closer to the original tool's functionality, but using the component would require the user to use `fromState` to split out the directory into the specific files.

**Comment by tverbeiren** [comment]
It's been some time since I looked at this (as visible from the date).

Am I right in saying:

- `output_prefix` points to a directory, e.g. `output_prefix="some/output/path"` ?
- `output_stats` and `output_dist` are relative to `output_prefix`, e.g. `stats_file` would end up under `some/output/path/stats_file` ?

If that is the case, the second option gets my preference:

- `output_prefix` is a label, used in the `Name` column and plots
- `output_stats` becomes a full path: `output_stats="/some/output/path/stats_file"`

The only caveat is that the labels do not match the paths, but I don't think that's a tenable position anyway.

WDYT?

**Comment by DriesSchaumont** [comment]
> `output_prefix` points to a directory, e.g. `output_prefix="some/output/path"` ?

No, it points to a directory _plus_ the prefix of the output files.

**Comment by tverbeiren** [comment]
And the directory + actual prefix are used as a label?

**Comment by DriesSchaumont** [comment]
> And the directory + actual prefix are used as a label?

Yes, e.g. when using `-o /elsewhere/nested/abspref` (according to Claude):

```
Name                       Mean               Median  sd
/elsewhere/nested/abspref  -64.5408163265306  -67.5   58.9042157460148
```
